web-registry/src/components/molecules/GettingStartedResourcesSection.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import { useMediaQuery, useTheme } from '@mui/material';
+
+import { GettingStartedResourcesCard } from 'web-components/lib/components/cards/GettingStartedResourcesCard';
+import Section from 'web-components/lib/components/section';
+import ResponsiveSlider from 'web-components/lib/components/sliders/ResponsiveSlider';
+import { Theme } from 'web-components/lib/theme/muiTheme';
+import { getLinkTarget } from 'web-components/lib/utils/linkTarget';
+
+import { GettingStartedResourcesSectionFieldsFragment } from 'generated/sanity-graphql';
+import { getBtnHref } from 'lib/button';
+import { getSanityImgSrc } from 'lib/imgSrc';
+
+import { Link } from 'components/atoms';
+
+const GettingStartedResourcesSection: React.FC<
+  React.PropsWithChildren<{
+    section: GettingStartedResourcesSectionFieldsFragment;
+  }>
+> = ({ section }) => {
+  const theme = useTheme<Theme>();
+  const isMobile = useMediaQuery(theme.breakpoints.down('sm'));
```

Comment:
Here we don't have the choice but to do a client check I think.

Reply:
indeed